jetty9/src/main/docker/setup-env-ext.bash

```diff
@@ -34,4 +34,9 @@ if ! type "$1" &>/dev/null; then
   export JAVA_OPTS="$JAVA_OPTS $JETTY_ARGS"
 fi
 
+# If we are deployed on a GAE platform, enable the gae module
+if [ -n "$GAE_INSTANCE" -a ! -e "$JETTY_BASE/start.d/gae.ini" ]; then
+  echo "--module=gae" > $JETTY_BASE/start.d/gae.ini
```

Comment:
Jetty's start.jar scans start.d to build it's command line at run time.
jetty.commands is used when building the image and has the ability to do --add-to-start which can download dependencies, create template config files and expand template ini files into start.d.

We could do everything by directly generating into start.d, but jetty.commands is more future proof as any future changes to configuration and/or dependencies will be picked up by the future call to --add-to-start.

In this case, I could have generated the ini file by running a start.jar --add-to-start=gae, but as it is our own module I didn't see the need for that extra level and just directly generated the ini file.